src/server/controller/cycleData/getTableData.ts

```diff
@@ -52,8 +55,9 @@ export const getTableData = async (
       const country = countryMap[countryIso]
       allTableNames.forEach((tableName) => {
         if (
-          tableName === TableNames.extentOfForest ||
-          (tableName === TableNames.forestCharacteristics && country.props.forestCharacteristics.useOriginalDataPoint)
+          tableName.toLowerCase() === TableNames.extentOfForest.toLowerCase() ||
+          (tableName.toLowerCase() === TableNames.forestCharacteristics.toLowerCase() &&
```

Comment:
why this check? it shouldn't be needed

Reply:
It shouldnt, but it is

Postgres changes the name to lowercase, and we use camelCase so it didnt match
![image](https://user-images.githubusercontent.com/5508251/192790078-f0449670-3302-4007-8d35-d3812dfca501.png)

Reply:
this must be restored as it was